Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -849,8 +761,8 @@ The default value of this LB map size is 65536.
 
 .. _kubeproxyfree_hostport:
 
-Container hostPort support
-**************************
+Container HostPort Support
```

Comment:
For reviewers: I was unsure of the proper case for `HostPort`. We seem to have a mix of `hostPort` and `HostPort` in our documentation, so I went with what was IMO less ugly. The K8s documentation also has a mix of both.

Reply:
Both are fine IMHO. `hostPort` was probably derived from the k8s obj spec.